### ISSUE TITLE
A duplicate video appears in Subscriptions -> All (probably related to premiere videos)

### ISSUE THREAD

**Issue by windows-server-2003** [TeamNewPipe/NewPipe#6933]
### Checklist

- [x] I am using the latest version - 0.21.8
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo.
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
This bug happened only once, so I cannot provide solid procedures to reproduce this.
However, what I think I did is as follows.

 1. Subscribe a channel that has an upcoming premiere video
 2. Reload Subscriptions -> All to include the video in the list
 3. Wait until the premiere starts (maybe ends ?)
 4. Reload Subscription -> All again.

### Actual behavior
The video loaded in 2. remains in the list and a new one appears when doing 4..

### Expected behavior
The premiere video should be in the list with updated metadata (such as the duration).

### Screenshots/Screen recordings
The duplicated items won't disappear and I can still see them.
<img src="https://user-images.githubusercontent.com/31173797/129573107-de8b6f9f-fd40-4f7f-aff5-514f2ba4eaa2.png" width="350">

### Logs
No logs

### Device info

 - Android version/Custom ROM version: Android 10
 - Device model: TCL Plex (T780H)

**Comment by SameenAhnaf** [comment]
Duplicate of #6072.

**Comment by windows-server-2003** [comment]
I saw the issue but overlooked that this issue is included in it. I'm sorry for not having checked thoroughly.
